apps/trading/pages/markets/trade-grid.tsx

```diff
@@ -199,6 +199,7 @@ export const TradeMarketHeader = ({
         <HeaderStat heading={t('Settlement asset')}>
           <div data-testid="trading-mode">
             <ButtonLink
+              className="inline underline hover:text-black dark:hover:text-white"
```

Comment:
Chore (blocking): like above, but this styling seems like something that could definitely be reused, so instead of customising this here, 

1. If you need an element to be inlined, consider using a wrapper tag with inline, rather than adding custom classes to an existing styled component.
2. Consider how to adjust the ButtonLink component

Reply:
In this instance, a styling change is required for hover text colour on the light theme because in the market header we use grey text for the values rather than black and the default text is the same grey, so I wanted to change the light theme hover text to black. The other styles (inline, underline etc) are only passed in because it seems the classNames property overwrites the default rather than appending. It only really applies to the market header and this is (currently) the only link there, so not sure how reusable we'd consider that?

If we do want to though, what'd be the best way?
I don't think wrapping would work, because the hover state would be overwritten by the child element.
In terms of adjusting ButtonLink, would I add a property to say whether it's being used in the header?

Reply:
Hmm, so the reason I thought that the 'hover:text-black dark:hover:text-white" bit could be reusable is for the times like you're describing where we want the text to be as vivid as possible. If it's genuinely single use for now and you don't anticipate it being used elsewhere (I can't think of anywhere that needs it either at the minute) then it's ok with me if it's kept as a one-off. 

Regarding using a wrapper - yeah you're correct about hover styles being overridden in the component, but I was just referring to it using the 'inline' class. Positional stuff like block/inline/grid styles/padding and margin etc can be applied to a parent wrapper, this keeps things cleaner and easy to manage moving forward.